tutorials/SpectroscopicDataReductionBasics/1-SpectroscopicTraceTutorial.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>I think as part of our style guide, we put all imports at the top.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I've moved this import to the top.</p>